schemas/2015-08-01/Microsoft.Web.json

```diff
@@ -374,7 +386,11 @@
                         }
                     }
                 }
-            }
+            },
+            "required": [
+              "name",
```

Comment:
There are no "type" or "apiVersion" properties on this object. Unless I misunderstand your comment?